tests/30rooms/12thirdpartyinvite.pl

```diff
@@ -53,6 +53,42 @@
       });
    };
 
+test "Can invite existing 3pid in createRoom",
+   requires => [ local_user_fixtures( 2 ), id_server_fixture() ],
+
+   do => sub {
+      my ( $inviter, $invitee, $id_server ) = @_;
+
+      my $invitee_mxid = $invitee->user_id;
+
+      my $room_id;
+
+      $id_server->bind_identity( undef, "email", $invitee_email, $invitee )
+      ->then( sub {
+         my $invite_info = {
+            medium    => "email",
+            address   => $invitee_email,
+            id_server => $id_server->name,
+         };
+         matrix_create_room( $inviter, invite_3pid => [ $invite_info ] )
+         ->then( sub {
```

Comment:
It would avoid the excessive indentation here, to end this `then` block and start another; by doing

       matrix_create_room( ... );
    })->then( sub {
       ( $room_id ) = @_;